jwst/cube_build/cube_build.py

```diff
@@ -874,10 +872,10 @@ def setup_IFUCube(self,j):
 
         IFUCube.meta.wcsinfo.wcsaxes = 3
 
-        IFUCube.flux_extension = 'SCI'
-        IFUCube.error_extension = 'ERR'
+        IFUCube.meta.flux_extension = 'SCI'
+        IFUCube.meta.error_extension = 'ERR'
         IFUCube.meta.dq_extension = 'DQ'
-        IFUCube.meta.weightmap = 'WMAP'
+        IFUCube.meta.weightmap_extension = 'WMAP'
         IFUCube.meta.data_model_type = 'IFUCubeModel'
```

Comment:
data_model_type is now defined in the core schema and automatically populated by the data model library when a model is created. So you don't need to set the value here (it can't hurt, it just isn't necessary). Also, you should remove the definition of data_model_type from the ifucube.schema.yaml (in datamodels/schemas/), because it's now already defined in core.schema.yaml which is inherited by ifucube.schema.yaml.